configure

```diff
@@ -68,251 +68,7 @@ echo "$SWIG_PATH" > tensorflow/tools/swig/swig_path
 # Invoke python_config and set up symlinks to python includes
 (./util/python/python_config.sh --setup "$PYTHON_BIN_PATH";) || exit -1
 
-## Set up Cuda-related environment settings
-
-while [ "$TF_NEED_CUDA" == "" ]; do
```

Comment:
So I am testing right now.

First issue: it wants to have CUDA, I think this should be made so we can avoid having it. I don't have CUDA on my machine, how can I avoid having it?

Second issue: I don't have the interface to select cuda from ./configure, it should set the corresponding env variable. I install cuda and it's on the path, it stills need to set the environment variable. Also it wants cudNN which I don't have on my machine so I am stopping there right now.